apidoc/Titanium/UI/iOS/Stepper.yml

```diff
@@ -48,28 +53,29 @@ properties:
 
   - name: wraps
     summary: |
-        If YES, incrementing beyond maximumValue sets value to minimumValue; likewise, decrementing below
-        minimumValue sets value to maximumValue. If NO, the stepper does not increment beyond maximumValue nor
-        does it decrement below minimumValue but rather holds at those values.
+        If YES, incrementing beyond <Ti.UI.iOS.Stepper.maximum> sets value to <Ti.UI.iOS.Stepper.minimum>. likewise, decrementing below
+        <Ti.UI.iOS.Stepper.minimum> sets value to <Ti.UI.iOS.Stepper.maximum>. If NO, the stepper does not increment beyond <Ti.UI.iOS.Stepper.maximum> nor
+        does it decrement below <Ti.UI.iOS.Stepper.minimum> but rather holds at those values.
         The default value is NO.
     type: Boolean
 
-  - name: minimumValue
-    summary: The miniumum value the stepper will be set to, the value must be smaller than the maximumValue.
-    type: Number
-
-  - name: minimumValue
-    summary: The maximum value the stepper will be set to, the value must be greater than the miniumumValue.
+  - name: minimum
+    summary: |
+        The miniumum value the stepper will be set to, the value must be smaller than the maximumValue.
+        If you attempt to set a value equal to or greater than maximumValue, the system will throw
+        an NSInvalidArgumentException exception.
```

Comment:
`maximumValue`, also the `NSInvalidArgumentException` is not thrown and should be caught on the native side. Same with the property below.